lib/pavilion/resolver.py

```diff
@@ -180,7 +180,10 @@ def filter_resolved_tests(self, resolved_tests, filters) -> List[ProtoTest]:
         filtered_tests = []
 
         for test in resolved_tests:
-            tests_dict[test[0]['suite'] + '.' + test[0]['name'] + '.' + test[0]['subtitle']] = test
+            if test[0]['subtitle']:
+                tests_dict[test[0]['suite'] + '.' + test[0]['name'] + '.' + test[0]['subtitle']] = test
+            else:
+                filtered_tests.append(ProtoTest(test[0], test[1].__deepcopy__()))
```

Comment:
Deep copying is what you want, but that's not how you do it.  Use `copy.deepcopy()`, or check that object's built-in `.copy()` function to see if it deep copies by default.